docs/tutorials/advanced-techniques-for-qaoa.ipynb

```diff
@@ -1265,7 +1264,7 @@
    "source": [
     "Finally, let's draw a graph based on the CVaR result.\n",
     "We split the graph nodes into two sets based on the CVaR result.\n",
-    "The nodes in the first set are colored in grey, and the nodes in the second set are colored in purple.\n",
+    "The nodes in the first set are colored in gray, and the nodes in the second set are colored in purple.\n",
```

Comment:
For accessibility we need a second way to distinguish between the two sets (not just the two colors). @beckykd suggestions?

Reply:
It depends on what the application is capable of.  Can they be pattern filled?  Can one be just an outline and one filled?